### ISSUE TITLE
APP: Gitea updated & Mycelium inside

### ISSUE THREAD

**Issue by xmonader** [threefoldtech/tfgrid-sdk-ts#2015]

**Comment by Mahmoud-Emad** [comment]
Blocked on https://github.com/threefoldtech/tf-images/issues/206

**Comment by zaelgohary** [comment]
@maxux Can you promote this too? https://hub.grid.tf/petep.3bot/gitea_mycelium.flist.md

**Comment by zaelgohary** [comment]
@ehab-hassan This too needs a logo, please.

**Comment by zaelgohary** [comment]
WIP:

Added the solution but didn't test it. Currently working on adding the env vars needed and testing.

**Comment by zaelgohary** [comment]
WIP:

Added Gitea to the deployment list. Also, removed the env tab and added mail server. However, when I tried deploying and log the services, one of them was in a state error. The issue is blocked till the flist is fixed.

**Comment by maxux** [comment]
Promoted: https://hub.grid.tf/tf-official-apps/gitea_mycelium.flist.md

**Comment by zaelgohary** [comment]
Work completed:

Edited flist.

**Comment by ehab-hassan** [comment]
@zaelgohary  logo white
![gitea](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/10484295/a96c7e88-f7e9-487c-94f6-17acbaee501d)

**Comment by khaledyoussef24** [comment]
tested on devnet
version : db2daf4
Users now can deploy a gittea solution from the dashboard.
but still need a domain to visit after creating the vm and needing also to add an icon to the deployment list to visit the domain 
and i think it's type of solutions which will have a control panel.
but the machine worked fine and the status of the tasks is running


![Image](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/110984055/1d7d3d4a-9415-4ec0-a62c-d6c7bc99c9b6)
![Image](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/110984055/b82c4f1c-85d7-499f-b217-bbc8c99a62a4)
![Image](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/110984055/9c0d4f0d-4091-4d05-954a-987923b0e4ad)
![Image](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/110984055/569559ef-8d42-4364-8c67-a5dc946b1392)

**Comment by Mik-TF** [comment]
Testing for docs on info_grid. Looks great so far! Nice work.

Some suggestions while testing:

- [ ]  Can we add a button like with Discourse: (Visit), leading to the Gitea instance? e.g. as we have with https://git.ourworld.tf/
![image](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/77026219/54f9b8ad-41cb-4129-aa6b-ae0d281373a7)
![image](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/77026219/cb144546-5fd9-48fb-a1d2-25f797510b92)
- [ ]  Can we add a possibility to set a domain (e.g. git.ourworld.tf leads to gitea instance of TF)
![image](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/77026219/09b36754-83eb-4c1f-b38c-adcb2074faa3)
- [ ]  Can we add the WireGuard option? I couldn't set it in Network and the wireguard configs aren't in the Gitea details screen

![image](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/77026219/a20fb8b0-aed2-4aa0-a04b-17405ca12c98)

**Comment by Mik-TF** [comment]
![image](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/77026219/3d0ed6d7-e603-4757-bebc-5064e1c95296)

Does this mean we cannot run Gitea without using IPv4 network?

**Comment by zaelgohary** [comment]
Work completed:

Added requested changes.

**Comment by zaelgohary** [comment]
@Mik-TF You can go on with the docs. Same goes for [Nostr](https://github.com/threefoldtech/tfgrid-sdk-ts/issues/2010).

**Comment by mik-tf** [comment]
Good! Will be done this week.

**Comment by khaledyoussef24** [comment]
verified and worked fine 
network used : devnet
version : ac03039


![Image](https://github.com/user-attachments/assets/50c9f548-85fb-4fd5-8a9c-f58f087477df)
![Image](https://github.com/user-attachments/assets/a9eb7086-f50a-465b-afbb-434e9fd29e7b)
![Image](https://github.com/user-attachments/assets/385da3ac-7e0f-47f6-bf23-b9fb827b15a2)
![Image](https://github.com/user-attachments/assets/80623462-7a75-4c44-bd45-adeec4f7c0ca)
![Image](https://github.com/user-attachments/assets/02049f62-7fed-4f40-a3cf-62df6a765e89)
![Image](https://github.com/user-attachments/assets/2facd9a5-8b29-4858-9683-2ac5aae886ce)
![Image](https://github.com/user-attachments/assets/68b6c4d8-dbb6-4ad0-a40f-a18863b449d6)
![Image](https://github.com/user-attachments/assets/a8d34e4f-790a-4cf8-940b-005323106100)

[testcase](https://app.testlodge.com/a/26076/projects/40893/suites/234374?expand_section=422332#case_4325396)

**Comment by mik-tf** [comment]
@khaledyoussef24 one thing is not clear.

![image](https://github.com/user-attachments/assets/ecfc9e3f-9248-4d27-8f9d-86c3dbcf1eb7)


It says SMTP is needed, and SMTP needs IPv4.

But I deployed a gitea instance without IPv4, and it still works. But I guess SMTP isn't set up.

Should we set IPv4 as true by default (and required) for the gitea app? Then SMTP will always work.
Otherwise, people will set Gitea instances without Ipv4 and then the instance won't work fully as they will not be able to receive emails.

Or perhaps when we don't set IPv4, it takes the IPv4 from the gateway? In that case, the warning (see screenshot above) is misleading.

Wdyt?

**Comment by mik-tf** [comment]
Work done on info_grid's side.

**Comment by zaelgohary** [comment]
> Work done on info_grid's side.

Gitea docs is returning 404.

![image](https://github.com/user-attachments/assets/b8210d02-8070-444e-85fa-70cd07ccfb01)


As for the SMTP warning, I'll take a look into it.

**Comment by mik-tf** [comment]
Of course, it's not yet merged: ongoing PR: https://github.com/threefoldtech/info_grid/pull/592

Great for the SMTP. Thanks!
